### PR TITLE
[4.0.0] Add gateway environment configuration for enforcer and router

### PR DESCRIPTION
**Pull request by renuka-fernando**
## Purpose
$subject

### CODE REVIEW

**Review by VirajSalaka — APPROVED** [review]

**Review by pubudu538 — APPROVED** [review]

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md [hunk 1]
```diff
@@ -12,8 +12,23 @@ Multiple gateway environments can be registered in the WSO2 API Manager control
 
 Let's assume you have two environments called "choreo-connect-1" and "choreo-connect-2".
 
+!!! Important
+    Even though the adapter has the capability to deploy APIs under multiple environment Labels (gateway labels), it is recommanded to apply for a single gateway environment label to maintain the deployment architecture simple.
```
[the remark this answers is not in the document]
Comment (samgnaniah) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md:16]: Check if this is ok @renuka-fernando

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md [hunk 2]
```diff
@@ -12,8 +12,23 @@ Multiple gateway environments can be registered in the WSO2 API Manager control
...
+
 1. In each of the Adapter, add the relevant environment to `environmentLabels` in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect-with-apim/conf/config.toml` under `controlPlane`.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md:18]:
```suggestion
1. In each of the Adapters, add the relevant environment to `environmentLabels` in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect-with-apim/conf/config.toml` under `controlPlane`.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md [hunk 3]
```diff
@@ -39,11 +54,35 @@ Let's assume you have two environments called "choreo-connect-1" and "choreo-con
         reconnectInterval = 5000
         reconnectRetryCount = 60
     ```
-        
-2. Start Choreo Connect deployment from docker-compose. 
 
-3. Add the Gateway environments "choreo-connect-1" and "choreo-connect-2" from the Gateways in the APIM admin portal.
+2. Configure gateway environment label in Enforcer and Router. Update the following environment labels in each component. Update the following environment variables defined in the docker-compose file.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md:58]:
```suggestion
2. Configure the Gateway environment label in the Enforcer and Router. Update the following environment labels in each component. Update the following environment variables defined in the docker-compose file.
```

Comment (samgnaniah) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md:58]: There are too many steps lumped in together here. Can we break this into multiple steps or reword this to make sure people know what they are doing here?

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md [hunk 4]
```diff
@@ -39,11 +54,35 @@ Let's assume you have two environments called "choreo-connect-1" and "choreo-con
...
+      ``` tab="Enforcer 1"
+      ROUTER_LABEL=choreo-connect-1
+      ```
+      ``` tab="Enforcer 2"
+      ROUTER_LABEL=choreo-connect-2
+      ```
+
+3. Start Choreo Connect deployment from docker-compose.
+
+4. Add the Gateway environments "choreo-connect-1" and "choreo-connect-2" from the Gateways in the APIM admin portal.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md:84]: Are these environments or labels?

Comment (renuka-fernando) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md:84]: environment labels, but in the admin portal below it is gateway environments

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md (hunk 3 above)
Comment (renuka-fernando) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md:58]:
```suggestion
2. Next let's configure the Gateway environment label in the Enforcer and Router. For that update the following environment variables in each service in the docker-compose file.
```
Yes we can keep this as a one step, reworded @samgnaniah could you please check this again?

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md [hunk 5]
```diff
@@ -39,11 +54,35 @@ Let's assume you have two environments called "choreo-connect-1" and "choreo-con
         reconnectInterval = 5000
         reconnectRetryCount = 60
     ```
-        
-2. Start Choreo Connect deployment from docker-compose. 
 
-3. Add the Gateway environments "choreo-connect-1" and "choreo-connect-2" from the Gateways in the APIM admin portal.
+2. Next let's configure the Gateway environment label in the Enforcer and Router. For that update the following environment variables in each service in the docker-compose file.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md:58]:
```suggestion
2. Next let's configure the Gateway environment label in the Enforcer and Router. To do that, update the following environment variables in each service in the docker-compose file.
```

**Review by samgnaniah — APPROVED** [review]
